spec/features/admin_user/admin_user_home_page_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.feature "the admin user" do
+  let(:valid_attributes) {
+    {full_name: "Admin Annie",
+      user_name: "admin",
+      password: "password",
+      role: 1,
+      email: "admin@admin.com"}
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside } missing.